test/ea_restaurant_data_loader_web/controllers/oauth2_controller_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule EaRestaurantDataLoader.Test.EaRestaurantDataLoaderWeb.Controllers.Oauth2ControllerTest do
+  use EaRestaurantDataLoaderWeb.ConnCase
+  alias EaRestaurantDataLoader.Test.Fixtures.AppClientFixture
+  alias EaRestaurantDataLoader.Test.Fixtures.UserFixture
+  alias EaRestaurantDataLoader.Test.Fixtures.AppClientScopeFixture
+  alias EaRestaurantDataLoader.Lib.Utils.ApplicationUtil
+  alias EaRestaurantDataLoader.Lib.Utils.Oauth2Util
+  alias EaRestaurantDataLoader.Lib.ErrorHandlers.InvalidCredentialsError
+  alias EaRestaurantDataLoader.Lib.ErrorHandlers.BadRequest
+
+  describe "oauth2 controller test" do
+    test "login - client credentials", %{conn: conn} do
```

Comment:
Start test saying `tes "should ..."`, i.e.  `tes "should login using client credentials"`

Reply:
Done.